### PR TITLE
changed texts on villain cards

### LINKED ISSUE #1173: Anathema,  Dynamo character cards refer to plural character cards (maybe affects most villains)
The backside of the Dynamo Character card, under Game Play, says "...flip Dynamo's Villain Character Card**s**" .  I think there is just the one character card?

### PR DESCRIPTION
**Pull request by cdallasanta**
closes #1173

I changed all occurrences of "Flip {villain}" to "{villain} flips" and "put {villain}'s character cards into play" to "{villain} enters play." There are a couple edge cases that could definitely use some eyes on it, and I'm sure I've left a typo or two in there.

### CODE REVIEW

**Comment by cdallasanta** [comment]
There are a few inconsistencies from the base game left, such as "when this card would be destroyed, {villain} flips instead," the base game says 'flips to the "Vengeful Mad Scientist" side instead.'

**Comment by cdallasanta** [comment]
@wrhyme29, it looks like I duplicated some of your past work. Can you make sure I didn't undo any of it? Also, is this change to "{villain} flips" instead of "flip the {villain} character cards" to match the base game a needed one?

**Comment by wrhyme29** [comment]
I believe that those PRs with what I did have already been merged with main, so unless you are using an outdated branch, you should be fine. 

The reason there is that difference in wording is that Cauldron is based on the original villain cards (where the setup and art cards are separate) whereas the video game is based on the oversized villain cards. This results in a lot of wording being multiple cards vs singular. I don't think it is a big deal to have them difference, but we certainly should be striving to match the base game as much as possible (with exceptions to the things it wouldn't make sense to match)

File: CauldronMods/DeckLists/Villain/TiamatDeckList.json [hunk 1]
```diff
@@ -139,7 +139,7 @@
             "gameplay": [
                 "{WinterTiamatCharacter} is immune to cold damage.",
                 "At the end of the villain turn, if {WinterTiamatCharacter} dealt no damage this turn, she deals the hero target with the highest HP {H - 2} cold damage.",
-                "When {WinterTiamatCharacter} is destroyed, flip her."
+                "When {WinterTiamatCharacter} is destroyed, her she."
```
Comment (LNBertolini) [inline CauldronMods/DeckLists/Villain/TiamatDeckList.json:142]:
```suggestion
                "When {WinterTiamatCharacter} is destroyed, flip her."
```

**Review by wrhyme29 — APPROVED** [review]

**Pull request merged**